docs/api/rebac/groups.mdx

```diff
@@ -6,60 +6,93 @@ title: Groups API
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
-The Groups API allows you to manage your resources and users within groups. You can create, update, delete, and list groups.
-Groups enable you to easily group together resources, resource-instances, and users. In addition groups simplify ReBAC permissions for users, by allowing roles to be assigned to the groups themselves.
-You can assign roles to a group and then add users to the group through the Groups API.
-The users within the group will derive roles form the roles assigned to the group .
+The Groups API simplifies relationship-based permission management for your resource instances and users by using groups. Groups, as suggested by the name, group together resource instances and users, enabling you to assign group members with permissions on resource instances.
 
-Check the [Groups Redoc ](/) for more information.
+## Key Features:
 
+- **Create, Update, Delete, and List Groups**: The Groups API provides a comprehensive set of endpoints to manage the lifecycle of your groups, ensuring you have complete control over their configuration and membership.
 
-## Example Usage for User Groups
 
-### Assign roles to a group:
+- **Assigning roles to Groups**: Groups can be assigned roles directly (instead of users). When adding users to a group through the API, you set them to automatically derive their permissions from the roles assigned to the group, 
+ensuring consistent and manageable access control. See the [Groups API Redoc](https://api.permit.io/v2/redoc#tag/Groups-(Upcoming-Feature)) for more information.
 
-Let's say we want to represent a company and its employees as a group, and then assign all the employees access to a shared marketing resource (social-media) via the group.  We'll have a group named 'company' (Which is represented as a resource named 'company'), and an instance of a specific company 'Acme'.  Users are assigned to `Acme` via a role called `member`
-Additionally, we will have a resource-instance of type 'marketing' - 'social_media' which supports the role 'editor'.
-Once we assign the role 'editor' to  the `company` group, the following will by automated by API:
-- create role derivation from 'company#member' to 'marketing#editor'
-- create a relation between 'company' and 'marketing' with a relation between 'company_group'
-- create relationship between 'company#Acme' and 'marketing#social_media'.
 
-![](groups-diagram.png)
+- **Group Resource Instances, and Users**: Organize your resource instances and users into groups, allowing for simplified ReBAC permissions for:
 
-Then we will have group 'company' and every user who is a 'member' in this group will have the role 'editor' in the resource 'marketing#social_media'.
+    - **Users:** Users: Including users in a group allows you to assign an entire cluster of users with a single role (Instead of assigning roles to each user individually). 
```

Comment:
```suggestion
    - **Users:** Including users in a group allows you to assign an entire cluster of users with a single role (Instead of assigning roles to each user individually). 
```